plugins/scaffolder-backend-module-gitlab/src/actions/gitlabGroupEnsureExists.ts

```diff
@@ -82,15 +79,15 @@ export const createGitlabGroupEnsureExistsAction = (options: {
         );
         if (!subGroup) {
           ctx.logger.info(`creating missing group ${fullPath}`);
-          parent = await api.Groups.create(
+          parent = (await api.Groups.create(
             pathElement,
             pathElement,
             parent
               ? {
-                  parent_id: parent.id,
+                  parentId: parent.id,
                 }
               : {},
-          );
+          )) as GroupSchema;
```

Comment:
Is there a reason for casting this?

Reply:
The `.create()` method returns an `ExpandedGroupSchema` which extends `GroupSchema` but isnt assisgnable.
I have refactored it to only use the `id` property, no need to keep the full `GroupSchema` around.